docs/data/toolpad/core/components/account/account.md

```diff
@@ -30,16 +30,45 @@ When signed out, the component renders as an inline sign in button within the da
 
 ## Customization
 
-### Slot Props
+The `<Account />` component is composed of multiple sub-components:
+
+- `<SignInButton />`
+- `<AccountPreview />`
+- `<Popover />`
```

Comment:
This is the only one of these 4 components that's actually just a standard Material UI component, right?
Not sure if that can be confusing in any way.

Reply:
Yes, not sure if I need to clarify that in any way? Perhaps okay to leave it as it is and see if that inspires any confusion